src/connectors/quipuswap/utils/config/tokens.ts

```diff
@@ -0,0 +1,64 @@
+import { NetworkType, Standard, SupportedNetwork, Token } from "../shared/types";
```

Comment:
Rather than defining tokens this way, you should the existing Token List-based way of defining tokens for Tezos: https://hummingbot.org/gateway/tokens/#what-are-token-lists

If this DEX is available to users on various Tezos networks, it should not adopt a new standard for mapping symbols to tokens. You can update the existing Tezos spec as well: https://github.com/hummingbot/gateway/blob/main/src/templates/tezos.yml

Reply:
Ah yes, I will update this.